pypeit/core/flux_calib.py

```diff
@@ -612,6 +613,8 @@ def sensfunc(wave, counts, counts_ivar, counts_mask, exptime, airmass, std_dict,
     out_table = table.Table(meta={'name': 'Sensitivity Function'})
     # These are transposed because we need to store them in an astropy table, with number of rows = norders
     out_table['SENS_WAVE'] = wave_arr.T
+    # TODO: Is this correct?
```

Comment:
I believe so.

Reply:
To clarify, the reason I asked is because this wasn't being added to the table in the previous version of the code.  I added saving `SENS_COUNTS_PER_ANG` in an attempt to unify the datamodel between the `SensFunc` subclasses.  I just wasn't sure the `counts_arr` was the correct thing to assign.  I've removed the `TODO` comment, but let me know if you think this is being incorrectly assigned.

Reply:
I think this is correct and agree with making them consistent as you've done.